### DISCUSSION TITLE
implement wotlk classic new retribution paladin changes

### DISCUSSION THREAD

**Discussion by nicedraeger** [chromiecraft/chromiecraft#4936] [Suggestions]
the changes recently announced by blizzard for wotlk classic, comprising a new glyph for hand of reckoning & changes to vengeance and seal of corruption/vengeance procs (able to proc via crusader strike & divine storm instead of just white swings), ought to be implemented on chromiecraft as well. perhaps effective in T7, perhaps earlier, but they seem worthy and welcome for a spec that truly 'falls off' for a while, until T10.

**Comment by balleny** [comment]
I would recommend to add a link to the retail change log, spell ids etc.

Most likely, this would be a change to the CC progression approach tho, not to AzerothCore - but that may change also until we reach WotLK content.

(IMO, we would need some balancing changes, esp. for paladins in pvp - and the adjustments blizzard is taking, may be or not suitable for CC at TBC, too)

**Comment by w5860363** [comment]
https://www.wowhead.com/wotlk/spell=405003/glyph-of-reckoning#used-by-item
https://www.wowhead.com/wotlk/item=204385/%E6%B8%85%E7%AE%97%E7%AC%A6%E6%96%87

https://us.forums.blizzard.com/en/wow/t/upcoming-adjustments-to-retribution-paladins/1511714

**Comment by trinitarius** [comment]
Even though I think balancing classes/races is in general a good thing for private servers, it is a lot of work to do it fair for everyone and only one buff to one spec will not result in balanced gameplay as PvP and PvE have to be considered for balancing. I think CC should not blindly copy Blizzard's custom changes from classic but rather think about doing a balancing module on their own if they have the time and staff for that.

During the 60 bracket feral DPS was "unplayable", on average it was dealing as much damage as the Prot Warrior tank. It surely deserved a minor buff to make it at least viable to play back then. A balancing module combined with the progression module would be a good idea.

**Comment by Never6** [comment]
Ret will be fine in t7 cause seal damage is still broken
